app/uk/gov/hmrc/lisaapi/controllers/Responses.scala

```diff
@@ -96,8 +96,6 @@ case object ErrorApiNotAvailable extends ErrorResponse(403, "API_NOT_AVAILABLE",
 
 case object ErrorUnauthorized extends ErrorResponse(401, "UNAUTHORIZED", "Bearer token is missing or not authorized")
 
-case object ErrorNotFound extends ErrorResponse(404, "NOT_FOUND", "Resource was not found")
```

Comment:
I'm guessing this was just a case of these objects not being used?

Reply:
Correctamundo